scripts/setup-test-db.sh

```
@@ -0,0 +1,122 @@
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+echo -e "${GREEN}========================================${NC}"
+echo -e "${GREEN}テスト用データベースのセットアップ${NC}"
+echo -e "${GREEN}========================================${NC}"
+echo ""
+
+# 環境変数の読み込み
+if [ -f .env ]; then
+    echo -e "${GREEN}✓${NC} .env ファイルを読み込んでいます..."
+    export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)